src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -334,13 +352,28 @@ void FullyConnected::createPrimitive() {
         return;
     }
 #endif
+
+#ifdef OV_CPU_WITH_LLMDNN
+    if (stateLLMFc != State_NotUse) {
+        if (initLLMFc()) {
```

Comment:
This initLLMFC checks whether it's suitable to use `LLMFC`, why not put this in `getSupportedDescriptors` just like MLAS does.

Reply:
Intrinsic library will assemble the needed parameters when doing the checks. If separating the check and extraction there will be some similar code, so putting them into one function should be cleaner. As name implied, put the `init` function into `createPrimitive` should be better than `getSupportedDescriptors`.